lib/modules/manager/helmv3/artifacts.ts

```diff
@@ -218,3 +221,12 @@ export async function updateArtifacts({
     ];
   }
 }
+
+function isHelmLockChanged(oldContent: string, newContent: string): boolean {
+  const regex = regEx(/^generated: ".+"$/gm);
```

Comment:
```suggestion
  const regex = regEx(/^generated: ".+"$/m);
```
i don't think you need the global flag, there is only one occurrence of it.